server.py

```diff
@@ -50,8 +50,168 @@ def return_items_needed_by_given_foodbank (name_of_foodbank): # Give a foodbank
 
 class index:
 	def GET(self):
-		return "Hello! This is an API that gives information on nearby foodbanks and the items they need. Check out the documentation, when it exists..."
+		return "Hello! This is an API that gives information on nearby food banks and the items they need. Check out the documentation, when it exists..."
 
+class documentation:
+	def GET(self):
+		return 
```

Comment:
I suspect this'll need to be a multi-line string (start and finish the string with """)